enumerate_iam/main.py

```diff
@@ -319,6 +325,15 @@ def enumerate_role(iam_client, output):
         for policy in role_policies['AttachedPolicies']:
             logger.info('-- Policy "%s" (%s)', policy['PolicyName'], policy['PolicyArn'])
 
+            get_policy = iam.get_role_policy(PolicyName=policy['PolicyName'])
+            policy_version = iam_client.get_policy_version(PolicyArn=policy['PolicyArn'], VersionId=policy['DefaultVersionId'])
```

Comment:
Maybe do some error handling here? The user might not have permissions to read one of the policies / policy versions, but have permissions to read the next one in `role_policies['AttachedPolicies']`. We want to handle errors here to extract as much information as possible.

Reply:
You're right, I'll need to refactor these bits.